tobac/segmentation.py

```diff
@@ -120,6 +123,33 @@ def segmentation_timestep(field_in,features_in,dxy,threshold=3e-3,target='maximu
     else:                
         raise ValueError('unknown method, must be watershed')
 
+	#create isolation, currently only available for 2d tracking
+    if field_in.ndim==2:
```

Comment:
Do we want to make this entirely optional? Is running the isolation here necessary for everyone, or would it be better to have a parameter in place to turn it on/off?

Reply:
I guess the question here ultimately is: what's the performance penalty?